src/utils/BrambleExtensionLoader.js

```diff
@@ -59,7 +59,8 @@ define(function (require, exports, module) {
      * by adding the extension name to the enableExtensions query param.
      */
     var extraExtensions = [
-        "brackets-cdn-suggestions"    // https://github.com/szdc/brackets-cdn-suggestions
+        "brackets-cdn-suggestions",    // https://github.com/szdc/brackets-cdn-suggestions
+        "selfie-taker"
```

Comment:
Let's change the name of this to `ImageUrlCodeHints` to reflect that it does more than simply take selfies.